cdc/cdc/processor/pipeline/sorter.go

```diff
@@ -151,6 +151,30 @@ func (n *sorterNode) StartActorNode(ctx pipeline.NodeContext, eg *errgroup.Group
 						}
 					}
 
+					// We calculate memory consumption by PolymorphicEvent size.
```

Comment:
About `PerKeySpanMemoryQuota`:

1. The default value `10MB` may be too small. `1GB` may be fine.

2. I think we can change the config item name `per-keyspan-memory-quota` to `per-changefeed-memory-quota`, as users have no idea that what `keyspan` is.

3. It would better to default to a proportion of total system memory, for ease of use. Refer to [`MaxMemoryPresure`](https://github.com/tikv/migration/blob/b559233b0f166429b904b8a4390db310a966dacb/cdc/pkg/config/server_config.go#L88). (Not a must in this PR).

Reply:
Thanks for your suggestion, I do the following:
1. changed `10MB` to `1GB`;
2. changed `per-keyspan-memory-quota` to `per-changefeed-memory-quota`;